Casks/a/aladin.rb

```diff
@@ -1,8 +1,11 @@
 cask "aladin" do
+  arch arm: "arm", intel: "amd"
+
   version "12.060"
-  sha256 :no_check
+  sha256 arm:   "d196277ef3c2964d49c34f0ca4b393711470d71f67a253d7eb82674b2ca2c9c4",
+         intel: "4c03dfaebde6585b7c58cea8a442219bd387a9075ad7560028875cf0cf26b616"
```

Comment:
The download `url` is unversioned, so we can't use a checksum here.

```suggestion
  sha256 :no_check
```